test_util/run-all

```diff
@@ -113,8 +113,16 @@ RET=$?
 mkdir -p $WORKDIR/test/vagrant-logs
 if [ $RET -ne 0 ]; then
   echo "TEST FAILED"
+  echo "Attempting to gather logs"
+  export LOG_REPLAY_LINES=${LOG_REPLAY_LINES:=1000}
```

Comment:
This doesnt need to be configurable as much as it is set at 10000 in TC right now, and that seemed a bit excessive to hard code in